bftengine/src/bcstatetransfer/BCStateTran.hpp

```diff
@@ -167,6 +167,7 @@ class BCStateTran : public IStateTransfer {
                                  char* outStateDigest,
                                  char* outResPagesDigest,
                                  char* outRVBDataDigest);
+  void startCollectingStateExternal();
```

Comment:
What does collectingstateexternal means?

this function should either be renamed or documented, especially because one line after we have a similarly named function.